### PR TITLE
6pm-2 Add link to QA for 6pm-2

### PR DESCRIPTION
**Pull request by Sean-syf**
Deployed QA on Heroku. Added links in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Storybook is here:
 * Production: <https://ucsb-cs156-w22.github.io/team04-w22-5pm-courses-docs/>
 * QA:  <https://ucsb-cs156-w22.github.io/team04-w22-5pm-courses-docs-qa/>
 
+6pm-2
+* QA dashboard: <https://dashboard.heroku.com/apps/s22-6pm-2-courses-qa/>
+* QA app: <https://s22-6pm-2-courses-qa.herokuapp.com/>
```
Comment (andrewhlu) [inline README.md:11]: The code changes look good, but can you add Bryan and Kevin in as collaborators in the Heroku app?
When you're finished, mark this as resolved and re-request my review.

**Review by andrewhlu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sean-syf) [reply README.md:11]: Added As Collaborators

**Review by andrewpengucsb — APPROVED** [review]
LGTM!

**Review by mikeycurtis — APPROVED** [review]
LGTM

**Pull request merged**